downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -461,6 +466,123 @@ public function main() returns error? {
+- Introduced new APIs to be consistent with MongoDB's native APIs.
+- Added support for MongoDB aggregation operations.
+- Added support for MongoDB projections through type inference and manual projection.
+- Added support for connecting to MongoDB Atlas databases and clusters.
+- Added support for SSL connections to MongoDB servers.
+- Improved the `find` API with support for filtering, sorting, and pagination.
+- Improved the `update` API by providing `matchedCount` and `modifiedCount` in the response.
+
+#### `redis` package
+
+- Added support to connect and work with Redis clusters.
+- Introduced support for secure connections (SSL/TLS) to Redis servers.
+- Added support for Redis connection strings (i.e. Redis URIs).
+- Extended connector compatibility to include the latest Redis server versions (up to 7.2.x).
+
+#### `zendesk` package
+
+- Introduced support to connect to Zendesk REST API V2, combining the functionalities of the `zendesk.support` and `zendesk.voice` packages.
+
+#### `rabbitmq` package
+
+- Removed the previously deprecated `rabbitmq:Message` record. Consequently, corresponding APIs no longer accommodate this record. Users are advised to transition to utilizing subtypes of `rabbitmq:AnydataMessage` for continued functionality.
+
+#### `nats` package
+
+- Removed the previously deprecated `nats:Message` record. Consequently, corresponding APIs no longer accommodate this record. Users are advised to transition to utilizing subtypes of `nats:AnydataMessage` for continued functionality.
+
+#### `googleapis.gmail` package
+
+- Enhanced connector APIs by incorporating resource function syntax, along with improved documentation and examples.
+
+#### `twilio` package
+
+- Enhanced integration with Twilio services (messaging, voice calls, media services, etc.) offered via the Twilio Basic REST API version 2010-04-01. 
+
+#### `salesforce` package
+
+- Introduced client credentials flow and username-password flow options to the Salesforce connector.
+- Expanded support to include Bulk v2 API, APEX REST API, and additional REST API functionalities. 
+- Updated compatibility with the latest Salesforce REST API version (v59).
+- Improved documentation and examples.
+
+#### `aws.redshift` package
+
+- Introduced seamless connectivity to Amazon Redshift databases.
+
+#### `snowflake` package
+
+- Removed the `requestGeneratedKeys` option from the Client configuration. Given that Snowflake databases do not return generated keys, the `requestGeneratedKeys` option is now defaulted to `NONE`.
+
+#### `aws.dynamodb` package
+
+- Introduced new APIs to work with database backups with DynamoDB REST APIs.
+
+#### `aws.dynamodbstreams` package
+
+- Introduced a new connector to work with DynamoDB streams.
+
+#### `aws.sns` package
+
+- Enhanced the connector APIs to support all the APIs provided by the AWS SNS REST API.
```

Comment:
It would be better if we could organize the connectors alphabetically here. (It seemed this was ordered alphabetically previously, but now there are connectors that doesn't follow this order).

Reply:
@ThisaruGuruge agreed. I sent the initial PR with alphabetically ordered entries, which seems not preserved with the subsequent PRs. Reordered with https://github.com/ballerina-platform/ballerina-dev-website/pull/8908/commits/77bd17ff9ff93e45abec65497286d9218185a846